ftw/linkchecker/command/checking_links.py

```diff
@@ -44,34 +42,30 @@ def setup_plone(app, site_info):
     setSite(plone_site)
 
 
-def get_broken_relations_in_plone_page():
-    relation_catalog = getUtility(ICatalog)
-    catalog_lookup(relation_catalog)
-    # there's need to also check another catalog
+def get_broken_relations_and_links():
+    portal_catalog = api.portal.get_tool('portal_catalog')
+    brains = portal_catalog.unrestrictedSearchResults()
+    site_links_and_relations = []
+    for brain in brains:
+        site_links_and_relations.extend(find_links_on_brain_fields(brain))
 
+    external_links = [[x[0], x[1], x[2]] for x in site_links_and_relations
```

Comment:
That's not readable at all.

Reply:
I opened an issue, so we can go on here #28